app/models/consignment.rb

```diff
@@ -20,6 +20,10 @@ class Consignment < ApplicationRecord
   validate :validate_user_roles
   before_save :upcase_bundle_consignment_seria
 
+  scope :search, lambda { |search|
+                   where("consignment_seria ILIKE '#{search.split[0]}%' and consignment_number::text  ILIKE '#{search.split[1]}%'")
+                 }
+
```

Comment:
не очень хорошая реализация у тебя идет split в контекстном поиске получается сравнение с null ?
если уже вообще по другому сделать нельзя то хотя бы на подзапросы разбей или выноси работу с параметров в контроллер

```suggestion
  scope :search, lambda { |search|
                    seria, number = search.split
                   query = "consignment_seria ILIKE '#{seria}%'"
                    query += " and consignment_number::text  ILIKE '#{number}%'" if !number.blank?
                    where(query)
                 }

```